build-tools/pnpm-lock.yaml

```diff
@@ -7441,7 +7413,7 @@ snapshots:
       eslint-plugin-react-hooks: 4.6.2(eslint@8.57.0)
       eslint-plugin-tsdoc: 0.2.17
       eslint-plugin-unicorn: 48.0.1(eslint@8.57.0)
-      eslint-plugin-unused-imports: 3.0.0(@typescript-eslint/eslint-plugin@6.7.5(@typescript-eslint/parser@6.7.5(eslint@8.57.0)(typescript@5.4.5))(eslint@8.57.0)(typescript@5.4.5))(eslint@8.57.0)
+      eslint-plugin-unused-imports: 3.0.0(@typescript-eslint/eslint-plugin@6.7.5(@typescript-eslint/parser@6.21.0(eslint@8.57.0)(typescript@5.4.5))(eslint@8.57.0)(typescript@5.4.5))(eslint@8.57.0)
```

Comment:
Now, this one... 😵‍💫 . I'm starting to think there's some non-determinism in the way pnpm resolves some of these peer deps, and the fact that for some reason there's a mix of 6.21.0 and 6.7.5, sometimes causes things like this. I wonder if updating our direct (in eslint-config-fluid and eslint-plugin-fluid) to 6.21.0 would help things.

Still, won't block on it.